sdk/src/wallet/operations/output_claiming.rs

```diff
@@ -204,22 +204,7 @@ where
         I::IntoIter: Send,
     {
         log::debug!("[OUTPUT_CLAIMING] claim_outputs");
-        let prepared_transaction = self.prepare_claim_outputs(output_ids_to_claim).await.map_err(|error| {
-            // Map InsufficientStorageDepositAmount error here because it's the result of InsufficientFunds in this
-            // case and then easier to handle
-            match error {
-                crate::wallet::Error::Block(block_error) => match *block_error {
-                    crate::types::block::Error::InsufficientStorageDepositAmount { amount, required } => {
-                        crate::wallet::Error::InsufficientFunds {
-                            available: amount,
-                            required,
-                        }
-                    }
-                    _ => crate::wallet::Error::Block(block_error),
-                },
-                _ => error,
-            }
-        })?;
```

Comment:
No mapping needed anymore?

Reply:
Well, the thing is that `InsufficientStorageDepositAmount` is never actually constructed